posthog/models/feature_flag.py

```diff
@@ -27,8 +23,10 @@ class FeatureFlag(models.Model):
     class Meta:
         constraints = [models.UniqueConstraint(fields=["team", "key"], name="unique key for team")]
 
-    name: models.CharField = models.CharField(max_length=400)
-    key: models.CharField = models.CharField(max_length=400)
+    key: models.CharField = models.CharField(max_length=64)
```

Comment:
any reason for the shorter max_lenght? Might cause some confusion/breaking behaviour.